JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,22 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  const HOURS_IN_SECONDS = 3600;
+  const hours = seconds / HOURS_IN_SECONDS;
+  const roundedHours = Math.floor(hours);
+
+  const min = (hours - roundedHours) * 60;
+  const roundedMin = Math.floor(min);
```

Comment:
for consistency and readability, try to keep the same convention on the variables names. In this case, keep the plural or the singular naming